app/representers/api/podcast_representer.rb

```diff
@@ -68,6 +68,13 @@ class Api::PodcastRepresenter < Api::BaseRepresenter
     } if represented.id
   end
 
+  link :guid do
+    {
+      href: api_podcast_guid_path_template(podcast_id: represented.id.to_s, id: '{guid}'),
```

Comment:
I opted to only add to podcast representers (not the API root).  And to call the param `guid` instead of `id`.  But I have no strong opinion.

Shows up something like:

![image](https://user-images.githubusercontent.com/1410587/210663036-2dc11a6d-7b14-4288-8470-e69414cae06e.png)

Reply:
Update: okay, I added to the API root and the Authorization root.  Because it seemed useful as I was writing some docs.